HACKING.md

```diff
@@ -17,7 +17,9 @@ This document gives a detailed breakdown of the various build processes and opti
 
 ## Build Pre-Requisites
 
-To build this project you must first install several command line utilities and a Kubernetes or OpenShift cluster
+To build this project you must first install several command line utilities and a Kubernetes or OpenShift cluster.
+
+You may want to try to use a ready-made Docker image based on CentOS 7 with all the tools pre-installed to quickly perform a build. See the instructions [here](HACKING-cli-image.md).
```

Comment:
This give the idea the Kubernetes or OpenShift clusters are included which is not true. MAybe you can split the first sentence into two parts and make it more clear what does this offer.